### PR TITLE
[docs]: How to fork a repo.

### PR DESCRIPTION
**Pull request by nisarhassan12**
Fixes gitpod-io/website#249

### CODE REVIEW

File: src/docs/fork-a-repo.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Fork a Repository
+
+Forking a repository in Gitpod is a simple 3 step process:
+
+- Open the _command pallete_ by either pressing `F1` or selecting `View --> Find Command...` from the menu in the top left corner as shown:
```
Comment (JesterOrNot) [inline src/docs/fork-a-repo.md:5]: Consider using an ordered list rather than bullet points

**Comment by nisarhassan12** [comment]
In the workspace, the build does not seem working:
image

**Comment by JesterOrNot** [comment]
I noticed I opened a preview split instead.

**Comment by nisarhassan12** [comment]
> I noticed I opened a preview split instead.

Thanks! @JesterOrNot Sorry I'm unable to understand what you mean? Can you please explain Thanks ^.^ ?

**Pull request merged**